hoomd/mpcd/geometry.py

```diff
@@ -56,6 +56,83 @@ def __init__(self, no_slip):
         self._param_dict.update(param_dict)
 
 
+class ConcentricCylinders(Geometry):
+    r"""Concentric-cylinders channel.
+
+    Args:
+        inner_radius (float): Radius of inner cylinders.
+        outer_radius (float): Radius of outer cylinders
+        angular_speed (float): Angular speed of the outer cylinder in the
+            counterclockwise direction.
+        no_slip (bool): If True, surfaces have no-slip boundary condition.
+            Otherwise, they have the slip boundary condition.
+
+    `ConcentricCylinders` confines particles between two cylinders
+    centered around the origin. The inner cylinder are stationary, but the
+    outer cylinder may put into motion with 'angular_speed'.
```

Comment:
Light edits here, and watch out that you want to use backtick not single quote to cross-reference in sphinx.
```suggestion
    centered around the origin. The inner cylinder is stationary, but the
    outer cylinder can rotate counterclockwise about the *z* axis with
    constant `angular_speed`.
```